src/connection/mod.rs

```diff
@@ -51,6 +51,17 @@ impl LogicalUnit {
     }
 }
 
+#[derive(Copy, Clone, Debug, PartialEq)]
```

Comment:
I'd prefer if we could move these three new things into `request.rs` and `pub use` them in this module. They may become less request specific in the future, but that is where we are using them now so having it a bit more centralized is good.